apis/r/R/TileDBGroup.R

```diff
@@ -365,21 +388,27 @@ TileDBGroup <- R6::R6Class(
     # list with names: name, uri, and type.
     get_all_members_uncached_read = function(group_handle) {
 
-      count <- tiledb::tiledb_group_member_count(group_handle)
+      #count <- tiledb::tiledb_group_member_count(group_handle)
+      count <- c_group_member_count(group_handle)
       if (count == 0) return(list())
 
       members <- vector(mode = "list", length = count)
       if (count == 0) return(members)
 
-      for (i in seq_len(count)) {
-        members[[i]] <- setNames(
-          object = as.list(tiledb::tiledb_group_member(group_handle, i - 1L)),
-          nm = c("type", "uri", "name")
-        )
-      }
+      #for (i in seq_len(count)) {
+      #  members[[i]] <- setNames(
+      #    # returns character vector with <type, uri, name>
+      #    object = as.list(tiledb::tiledb_group_member(group_handle, i - 1L)),
+      #    nm = c("type", "uri", "name")
+      #  )
+      #}
 
       # Key the list by group member name
-      names(members) <- vapply_char(members, FUN = getElement, name = "name")
+      #names(members) <- vapply_char(members, FUN = getElement, name = "name")
+
+      ## names list of lists
+      members <- c_group_members(group_handle)
+
```

Comment:
Can we clean out the commented code?
```suggestion
      ## names list of lists
      members <- c_group_members(group_handle)
```